test/terra/backends/qasm_simulator/qasm_set_state.py

```diff
@@ -120,3 +121,51 @@ def test_set_stabilizer(self, num_qubits):
             self.assertIn(save_label, data)
             value = qi.Clifford.from_dict(result.data(0)[save_label])
             self.assertEqual(value, target)
+
+class QasmSetMPSTests:
```

Comment:
Can you also add a test for the AerSimulator backend, you can see the other set state tests for how these should be run. It should looks something like:

```python
@supported_methods(['automatic', 'matrix_product_state'])
    def test_set_matrix_product_state(self, method, device):
        # test...
```

Reply:
@chriseclectic , I am not sure what to do here. Should I copy the test from qasm to `test/terra/backends/aer_simulator/instructions/test_set_state.py`?
Isn't this redundant?

Reply:
Done.